contracts/toCKB-typescript/src/switch/mod.rs

```diff
@@ -132,6 +136,34 @@ fn get_deletion_tx_type(data: &ToCKBCellDataView) -> Result<TxType, Error> {
     }
 }
 
+fn verify_xt(tx_type: &TxType) -> Result<(), Error> {
+    use TxType::*;
+    match tx_type {
+        DepositRequest
+        | Bonding
+        | WithdrawPledge
+        | WithdrawCollateral
+        | WithdrawPledgeAndCollateral
+        | LiquidationSignerTimeout
+        | LiquidationUndercollateral
+        | LiquidationFaultyWhenWarranty
+        | LiquidationFaultyWhenRedeeming => forbid_mint_xt(),
```

Comment:
It is better to use `forbid_mint_xt` as default behavior, or explicitly match all types for secure.
If we add new TxType here, it is easy to forget this and leave the new type unchecked.

Reply:
good idea